scrapper.py

```diff
@@ -48,14 +74,33 @@ def prepare_environment(base_path):
     """
     Creates ASSETS_PATH folder if not created and removes existing folder
     """
-    pass
+    try:
+        os.rmdir(base_path)
+    except FileNotFoundError:
+        os.mkdir(base_path)
 
 
 def validate_config(crawler_path):
     """
     Validates given config
     """
-    pass
+    with open(crawler_path, "r", encoding="utf-8") as file:
+        config = json.load(file)
+    seed_urls = config["seed_urls"]
+    number_of_articles = config["total_articles_to_find_and_parse"]
+    if not seed_urls or not isinstance(seed_urls, list):
+        raise IncorrectURLError
+    for seed_url in seed_urls:
+        if not isinstance(seed_url, str):
+            raise IncorrectURLError
+        norm_url = re.match(r"https?://", seed_url)
+        if not norm_url:
+            raise IncorrectURLError
```

Comment:
would you accept such links? https://github.com/fipl-hse/2021-2-level-ctlr/pull/24/files

should you?